server/spec/services/rpc_client_spec.rb

```diff
@@ -9,7 +9,7 @@ def publish_response(id, response)
   end
 
   def publish_error(id, error)
-    MongoPubsub.publish(channel, {message: [1, id, error, nil]})
+    MongoPubsub.publish("#{channel}:#{id}", {message: [1, id, error, nil]})
```

Comment:
This was causing the error spec to actually fail with a `RpcClient::TimeoutError` before.